feat: implement new design of help page

vzhukovs opened the pull request:
### What does this PR do?
Current changes proposal provides the new implementation of help page according to the latest mockups.

### Screenshot/screencast of this PR
Medium and large screen resolution:
![Monosnap Podman Desktop 2023-03-18 16-52-08](https://user-images.githubusercontent.com/1968177/226113429-258c24f6-d586-47be-bae7-7c3c63e2d7ca.jpg)

Also for small screen resolution:
![Monosnap Podman Desktop 2023-03-18 16-52-41](https://user-images.githubusercontent.com/1968177/226113439-e45884a7-aae0-4ada-9a8a-4294aed3e295.jpg)

### What issues does this PR fix or reference?
#431 

### How to test this PR?

Open the help page from the bottom right corner.

benoitf commented on packages/renderer/src/lib/help/HelpPage.svelte:
```diff
@@ -1,91 +1,187 @@
+      <div class="bg-zinc-800 px-3 pt-3 pb-3 rounded-lg">
+        <div class="text-lg">Core Podman Engine</div>
+        <div class="flex sm:flex-col md:flex-row py-3 md:h-24">
+          <div class="sm:h-full sm:my-auto md:flex md:flex-col md:w-1/2 md:border-r-2">
+            <div class="sm:mt-2 md:my-auto">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://podman.io/getting-started/')}"
+                title="https://podman.io/getting-started/">
+                Getting started with containers<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+            <div class="sm:mt-2 md:my-auto">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://docs.podman.io/en/latest/Commands.html')}"
+                title="https://docs.podman.io/en/latest/Commands.html">
+                View podman commands<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+          </div>
+          <div class="md:flex md:flex-col md:w-1/2">
+            <div class="sm:mt-2 md:my-auto md:text-right">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://podman.io/getting-started/installation')}"
+                title="https://podman.io/getting-started/installation">
+                Set up podman<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+            <div class="sm:mt-2 md:my-auto md:text-right">
+              <p
+                class="generic-link text-gray-100 cursor-pointer hover:no-underline"
+                on:click="{() => window.openExternal('https://docs.podman.io/en/latest/Tutorials.html')}"
+                title="https://docs.podman.io/en/latest/Tutorials.html">
+                View all tutorials<i class="fas fa-solid fa-external-link-alt ml-2" aria-hidden="true"></i>
+              </p>
+            </div>
+          </div>
+        </div>
```
These links are supposed to be brought by the extensions so it can't be hardcoded.
(it's like the link that we see on each provider)

vzhukovs replied:
updated, now links fetches from the contributions

benoitf reviewed (changes requested):
podman links should come from the extension, not from the core

Only Podman Desktop can be 'hardcoded'

vzhukovs commented on packages/renderer/src/lib/help/HelpPage.svelte:
```diff
@@ -1,91 +1,187 @@
+.generic-link:hover > i {
+  opacity: 100%;
+}
+</style>
```
updated, now opacity values take from tailwind css

vzhukovs commented:
Updated PR and reworked a bit to allow contributions provide help links. Below there is an example how links are provided from the contribution:

![contributed_links](https://user-images.githubusercontent.com/1968177/229606668-1fb2bb66-8b40-429f-9927-e9f678b76646.gif)


To display the link on the Dashboard page it is enough to use [extensionApi.DashboardGroupLink](https://github.com/containers/podman-desktop/pull/1735/files#diff-c885162dd627b7cd812c83544b19b9b166179fec79ddbc13273fede7c2566791R464) as primary group, otherwise, value will be used as section name in Help page.

benoitf commented on packages/extension-api/src/extension-api.d.ts:
```diff
@@ -248,6 +248,7 @@ declare module '@podman-desktop/api' {
   export interface Link {
     title: string;
     url: string;
+    group: LinkGroupName;
```
shouldn't the group be optional ?

benoitf replied:
probably no group name then it's displayed on the dashboard. It's OK I think to have a default group if not specified

vzhukovs replied:
updated, now the group name is optional
